.eslintrc.js

```diff
@@ -1,12 +1,12 @@
 module.exports = {
-	extends: '@studiometa/eslint-config/prettier-es6',
-	overrides: [
-		{
-			files: ['gulpfile.js', 'webpack.config.js'],
-			rules: {
-				'global-require': 'off',
-				'import/no-extraneous-dependencies': 'off',
-			},
-		},
-	]
-};
\ No newline at end of file
+  extends: '@studiometa/eslint-config',
+  overrides: [
```

Comment:
We can remove the overrides, the `gulpfile.js` and `webpack.config.js` files are not in the project anymore.